frontend/src/pages/ConfigurationPage.tsx

```diff
@@ -22,13 +24,27 @@ const RoadmapConfigurationPageComponent = ({
 }: {
   userInfo: UserInfo;
 }) => {
+  const { t } = useTranslation();
   const roadmapId = useSelector(chosenRoadmapIdSelector);
   const dispatch = useDispatch<StoreDispatchType>();
   const userType = getType(userInfo, roadmapId);
-
+  const [patchRoadmap] = apiV2.usePatchRoadmapMutation();
   const { data: integrations } = apiV2.useGetIntegrationsQuery(
     roadmapId ?? skipToken,
   );
+  const { data: roadmaps } = apiV2.useGetRoadmapsQuery();
+  const roadmap = roadmaps?.find(({ id }) => id === roadmapId);
```

Comment:
```suggestion
  const { data: roadmap } = apiV2.useGetRoadmapsQuery(undefined, {
    skip: !roadmapId,
    ...selectById(roadmapId),
  });
```

Could select by id directly from the query, and skip when no id is present.